packages/language-service/src/lib/provider.ts

```diff
@@ -766,6 +769,7 @@ export class Provider {
             (ps.selector[0] as SelectorChunk).classes[0] ||
             (ps.selector[0] as SelectorChunk).customSelectors[0] ||
             chunkStrings[0];
+        // transforms inline custom selectors (e.g. ":--custom" -> ".x .y")
```

Comment:
same comment

Reply:
I added these comments because it's really hard to understand what's going on in the old parts of the language service. And even with the specific code to "transformCustomSelectorInline"  I prefer to have the comments there to explain this.